core/src/components/range/test/range-events.e2e.ts

```diff
@@ -99,31 +107,6 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
 
         expect(rangeEndSpy.length).toBe(1);
       });
-
-      // TODO FW-2873
-      test.skip('should not scroll when the knob is swiped', async ({ page, skip }) => {
```

Comment:
Moved this test to it's own test page to properly test the scroll since the basic page didn't have enough content to scroll on.